tests/testthat/test-utils.R

```diff
@@ -0,0 +1,13 @@
+test_that("bad api key correctly fails", {
+  make <- "Mazda"
+  url_ <- "https://vrdb-tc-apicast-production.api.canada.ca/eng/vehicle-recall-database/v1/recall/make-name/"
+  url_ <- paste(url_, toString(make), "/count", sep = "")
+  expect_error(call_vrd_api(url_,make, api_key="test"))
+})
+
+test_that("call_rd_api() returns the correct structure",{
```

Comment:
tests the api call with out having to set an api key

there are a few default/test api urls outlined in the word doc describing api on the website